website/docs/tutorials/06-deploying/02-migrations/declare.md

```diff
@@ -2,17 +2,19 @@
 
 ```python
 def declare(
-    contract_path: str,
+    contract_path_str: str,
```

Comment:
If you are renaming this, could we remove it to `contract` now? #847 

Mind that renaming an argument is a breaking change in python due to kwargs semantics!

Reply:
The argument in the code isn't renamed. The doc was wrong.

Reply:
👍🏻